CHANGELOG.next.toml

```diff
@@ -16,3 +16,47 @@ message = "Add codegen version to generated package metadata"
 references = ["smithy-rs#1612"]
 meta = { "breaking" = false, "tada" = false, "bug" = false }
 author = "unexge"
+
+[[smithy-rs]]
+message = """
+Support granular control of specifying runtime crate versions.
+
+For code generation, the field `runtimeConfig.version` in smithy-build.json has been removed.
+The new field `runtimeConfig.verisions` is an object that keys are runtime crate names (e.g. aws-smithy-http),
+and values are their versions user specifying.
+
+If you previous set `version = "DEFAULT"`, then you're lucky.
+The new behaviour automatically detects and injects the version for you by checking the version of the code generator.
+You don't need to add any new configuration.
+
+If you specified a certain version such as `version = "0.47.0", you can migrate to a special reserved key `DEFAULT`.
```

Comment:
But they couldn't specify an explicit version before, right?

Reply:
I think they could

https://github.com/awslabs/smithy-rs/blob/0f47f69ddf76a41515ac2c198391cf080cd92b30/codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/RuntimeTypes.kt#L70-L80

Reply:
Ah, right. They could specify a concrete version, that's the `configuredVersion` branch.